templates/project_list.md

```diff
@@ -3,7 +3,7 @@
 5.  [Dash](/projects/Dash.md)
 6.  [NEM](/projects/NEM.md)
 7.  [Ethereum Classic](/projects/EthereumClassic.md)
-8.  [IOTA](/projects/IOTA.md)
+9.  [Monero](/projects/Monero.md)
```

Comment:
Strange change.  Was Monero missing last time?

Perhaps, just create issue for Monero and delete this file completely?